extensions/Cake.Addin3.yml

```diff
@@ -0,0 +1,16 @@
+Type: Addin
+Name: Cake.Addin3
+NuGet: Cake.Addin3
+Assemblies:
+- /**/Cake.Addin3.dll
+Repository: 
+ProjectUrl: https://www.nuget.org/packages/Cake.Addin3/
+Author: Daryl
+Description: TestPackage
```

Comment:
This should be blacklisted

Reply:
Adding this addin to exclusion list: : https://github.com/cake-contrib/Cake.AddinDiscoverer/issues/194